frontend/webpack.config.js

```diff
@@ -21,6 +21,39 @@ var browsersList = JSON.stringify(_.filter(browsersListConfig.split('\n'), funct
   return entry && entry.charAt(0) !== '#';
 }));
 
+var loaders = [
+  { test: /[\/]angular\.js$/,         loader: 'exports?angular' },
+  { test: /[\/]jquery\.js$/,          loader: 'expose?jQuery' },
+  { test: /[\/]moment\.js$/,          loader: 'expose?moment' },
+  { test: /[\/]mousetrap\.js$/,       loader: 'expose?Mousetrap' },
+  { test: /[\/]vendor[\/]i18n\.js$/,  loader: 'expose?I18n' },
+  {
+    test: /\.css$/,
+    loader: ExtractTextPlugin.extract(
+      'style-loader',
+      'css-loader!autoprefixer-loader?{browsers:' + browsersList + ',cascade:false}'
+    )
+  },
+  { test: /\.png$/,                   loader: 'url-loader?limit=100000&mimetype=image/png' },
+  { test: /\.gif$/,                   loader: 'file-loader' },
+  { test: /\.jpg$/,                   loader: 'file-loader' },
+  { test: /js-[\w|-]{2,5}\.yml$/,     loader: 'json!yaml' }
+]
+
+for (var k in pathConfig.pluginNamesPaths) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.